src/components/Project/ProjectsAPI.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://jsonplaceholder.typicode.com",
+});
+
+export default async function GetProjects() {
```

Comment:
Only functional components starts with uppercase, **getProjects** is the correct name.